packages/runtime/lib/app.js

```diff
@@ -107,14 +109,16 @@ class PlatformaticApp {
       })
     }
 
-    configManager.update({
-      ...configManager.current,
-      metrics: {
-        ...configManager.current.metrics,
-        defaultMetrics: { enabled: this.appConfig.entrypoint },
-        prefix: snakeCase(this.appConfig.id) + '_'
-      }
-    })
+    if (this.#hasManagementApi) {
```

Comment:
```suggestion
    if (this.configManager.current.managementApi) {
```

Reply:
Unfortunately, this is incorrect. The `managaementApi` is part of the runtime config, not the individual service config.